tests/src/performance/linkbench/linkbench.cs

```diff
@@ -272,107 +272,41 @@ public class LinkBench
                 () => Benchmark.AddLinkerReference("roslyn\\src\\Compilers\\CSharp\\csc\\csc.csproj"), true)
         };
 
-        static int UsageError()
+        public static int Main(string[] args)
         {
-            Console.WriteLine("Usage: LinkBench [--nosetup] [--nobuild] [--perf:runid <id>] [<benchmarks>]");
-            Console.WriteLine("  --nosetup: Don't clone and fixup benchmark repositories");
-            Console.WriteLine("  --nobuild: Don't build and link benchmarks");
-            Console.WriteLine("  --perf:runid: Specify the ID to append to benchmark result files");
-            Console.WriteLine("  --perf:outputdir: Specify the output directory used by xUnit Performance");
-            Console.WriteLine("    Benchmarks: HelloWorld, WebAPI, MusicStore, MusicStore_R2R, CoreFX, Roslyn");
-            Console.WriteLine("                Default is to run all the above benchmarks.");
-            return -4;
-        }
+            var options = BenchmarkOptions.Parse(args);
 
-        public static int Main(String [] args)
-        {
-            bool doSetup = true;
-            bool doBuild = true;
-            string runId = "";
-            string outputdir = ".";
-            string runOne = null;
```

Comment:
Was this variable never used? You remove it here, but it is no where else in the diff

Reply:
Yes, it was never used (maybe I should make the projects treat warnings as errors).

---
In reply to: [149472900](https://github.com/dotnet/coreclr/pull/14900#discussion_r149472900) [](ancestors = 149472900)